mlir/lib/Dialect/Rock/Transforms/BufferizableOpInterfaceImpl.cpp

```diff
@@ -90,9 +91,16 @@ struct GemmLikeInterface
     if (!outBuffer) {
       return op->emitOpError("Couldn't find output argument\n");
     }
+    if (op->getNumRegions() == 1) {
+      auto newOp = cloneWithoutRegions(
+          rewriter, op, /*newResultTypes=*/TypeRange{}, bufferArgs);
+      rewriter.inlineRegionBefore(op->getRegion(0), newOp->getRegion(0),
```

Comment:
Is this right? Don't we want to clone the region?

Reply:
I was just basing this off from : https://github.com/ROCm/rocMLIR/blob/4251df1dda5522598e5556179dd4359bd8f007c8/external/llvm-project/mlir/lib/Dialect/Linalg/Transforms/BufferizableOpInterfaceImpl.cpp#L78-L81

but you maybe right, I ll try if we can remove inlineRegionBefore...

Reply:
Huh, then that might just be it.